collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+
+Darío also implemented a new lightmapper denoising approach to replace old, extremely bulky and slow <abbr title="OpenImage Denoise">OIDN</abbr> denoiser. The new <abbr title="Joint Non-Local Means">JNLM</abbr> denoiser is much faster as it utilizes compute shaders (and thus your GPU), and it's also significantly more lightweight ([GH-81659](https://github.com/godotengine/godot/pull/81659)). While the quality difference can be noticeable under some circumstances, thanks to drastically reduced baking time you can now use higher resolution maps to counteract that.
+
+The team also worked on addressing multiple bugs in lightmapping quality and consistency ([GH-61910](https://github.com/godotengine/godot/pull/61910), [GH-81545](https://github.com/godotengine/godot/pull/81545), [GH-81872](https://github.com/godotengine/godot/pull/81872), [GH-81951](https://github.com/godotengine/godot/pull/81951), [GH-82533](https://github.com/godotengine/godot/pull/82533)).
+
+For Forward+ and Mobile rendering methods this release introduces 2D HDR rendering, unlocking 3D effects, such as glow, for 2D games ([GH-80215](https://github.com/godotengine/godot/pull/80215)). HDR rendering can be used to substantially improve quality of 2D rendering at the cost of performance. The Compatibility renderer also gets a new and anticipated feature — 3D shadows ([GH-77496](https://github.com/godotengine/godot/pull/77496)). Both of these changes were crafted by [Clay John](https://github.com/clayjohn).
+
+Last, but not least, an optional ANGLE-backed OpenGL renderer was added for macOS and Windows ([GH-72831](https://github.com/godotengine/godot/pull/72831)). ANGLE is a compatibility layer for OpenGL on top of Metal and Direct3D 11, which allows us to work around the deprecated and unmaintained OpenGL drivers on macOS, and similarly outdated OpenGL drivers on Windows for some older integrated chipsets. This should increase the portability of Godot games on lower end devices.
+
+#### Particles
+
+Our brilliant VFX champion [Ilaria Cislaghi](https://github.com/QbieShay) grew tired of messy implementation of GPU particles and organized a rework aimed at improving internal structures for better maintenance ([GH-79527](https://github.com/godotengine/godot/pull/79527)). Along the way, she also tweaked all sorts of bits and bobs to give VFX artists more control over particle motion. It's now possible to directly animate velocity over life, inherit projectile velocity, and change the emission amount of particles.
+
```

Comment:
```suggestion

Importantly, please note that with these changes the GPUParticles and CPUParticles will start to get out of sync. That is to say that we are no longer holding back features from GPUParticles that would be difficult to implement in CPUParticles. Now that all supported backends can support GPUParticles, we view CPUParticles as a low-cost, simplified alternative to GPUParticles, instead of a direct 1-to-1 equivalent implemented on the CPU. 

```

Reply:
I mention some of this in one for the follow-up paragraphs. I would like to avoid duplicating the same point twice so the piece doesn't bloat too much. Let me know if I can edit the text below to improve this message from your POV!